bananalyzer/__main__.py

```diff
@@ -87,6 +88,18 @@ def parse_args() -> Args:
         default=[],
         help="A list of ids to skip tests on, separated by commas",
     )
+    parser.add_argument(
+        "-q",
+        "--quiet",
+        action="store_true",
+        help="Will decrease the verbosity of pytest",
+    )
+    parser.add_argument(
+        "--turbo",
+        action="store_true",
+        help="Will run tests in a single browser instance. This is faster but less "
+        "reliable",
+    )
```

Comment:
What does less reliable mean? What are the failure modes? Why would I want to use this if it's not reliable?

Also, can we call this `--single_browser_context` or something more obvious? 

```suggestion
    parser.add_argument(
        "--single_browser_instance",
        action="store_true",
        help="Run tests in a single browser instance as opposed to creating a browser instance per test. This is faster but less reliable as test contexts will occasionally bleed into each other, causing tests to fail",
    )
```

Reply:
done